redis_cluster_arginfo.h

```diff
@@ -1267,11 +1269,6 @@ static zend_class_entry *register_class_RedisCluster(void)
 
 	INIT_CLASS_ENTRY(ce, "RedisCluster", class_RedisCluster_methods);
 	class_entry = zend_register_internal_class_ex(&ce, NULL);
-#if (PHP_VERSION_ID >= 80200)
```

Comment:
Are you accidentally removed this block?

Reply:
Good catch, will fix